### PR TITLE
Command re-send posts and signups to Quasar

### PR DESCRIPTION
**Pull request by katiecrane**
#### What's this PR do?
- New command to send posts and signups within a given date range (or all, if none given) to Quasar
    - Dates are based on the `updated_at` timestamps on the posts and signups
- Adds 3 tests for each scenario that the command can be run with (no arguments, start date given, and start and end date both given)

#### How should this be reviewed?
Does using `updated_at` make sense? Is it okay to assume we always want to send both posts and signups?

#### Relevant tickets
[Card](https://www.pivotaltracker.com/n/projects/2019429/stories/155196205)

#### How to deploy
https://github.com/DoSomething/rogue/blob/master/docs/development/deployments.md

### CODE REVIEW

File: app/Console/Commands/SendToQuasar.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        info('rogue:quasar - Starting to send to Quasar');
+
+        $start = $this->argument('start');
+        $end = $this->argument('end');
```
Comment (chloealee) [inline app/Console/Commands/SendToQuasar.php]: will this be defined is nothing is passed in as the `start` param?

File: app/Console/Commands/SendToQuasar.php [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+class SendToQuasar extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rogue:quasar
+                            {start=0000-00-00 : Include all signups and posts updated on or after this day. Format: YYYY-MM-DD}
```
Comment (sbsmith86) [inline app/Console/Commands/SendToQuasar.php]: lol is the default the beginning of time?

Comment (katiecrane) [reply app/Console/Commands/SendToQuasar.php]: haha yeah to grab all of them! I started out by checking if a `start` was provided or not, but it seemed like this way was going to be less code and easier to follow.

Comment (chloealee) [reply app/Console/Commands/SendToQuasar.php]: ah I see now!

File: app/Console/Commands/SendToQuasar.php (hunk 1 above)
Comment (katiecrane) [reply app/Console/Commands/SendToQuasar.php]: If only `start` (or no arguments) are passed, then `$end` gets set to `null`

**Pull request merged**